views/dashboard.handlebars

```diff
@@ -25,6 +25,42 @@
             <div class="cell medium-10 current-ads-label">
                 <h1>Current Ads:</h1>
             </div>
+            <!-- On first load need to reference the UserSettings table -->
+            <!-- set active the selection in the Value table -->
+            <!-- and onclick update the Value so that it will be set to that by default for that person's account -->
+            <!-- set up the filterSelection JS to take the different selections and store it then async load the page -->
+            <!-- check how Nate did his async load for disable and enable ads and see if I can copy that setting for the sort order -->
+            <!-- I don't really understand the Settings table tbh you only need the UserID and the button they selected (in this case the Value) -->
+            <!-- kind of a janky button set up but this is so we can click on it and see the results, can make it look pretty later -->
+            <!-- not sure how to get it so the buttons are all next to each other next to Current Ads header -->
+                <div class="cell medium-1" id="sortingContainer">
+     <!--       <button class="button active" onclick="filterSelection('a.Title ASC')"> Ascending Title </button>
+                <button class="button" onclick="filterSelection('a.Title DESC')"> Descending Title </button>
+                <button class="button" onclick="filterSelection('a.DatePosted ASC')"> Most Recent </button>
+                <button class="button" onclick="filterSelection('a.DatePosted DESC')"> Least Recent </button>
+                <button class="button" onclick="filterSelection('a.LocationRadius ASC')"> Ascending Location Radius </button>
+                <button class="button" onclick="filterSelection('a.LocationRadius DESC')"> Descending Location Radius </button>
+                <button class="button" onclick="filterSelection('a.DateCreated ASC')"> Ascending Date Created </button>
+                <button class="button" onclick="filterSelection('a.DateCreated DESC')"> Descending Date Created </button> -->
+
+<!-- the stuff above was just me playing around with function and buttons, can be disregarded or deleted -->
+<!-- below is the actual form for sorting ads based on the filter, currently there's nothing that shows which filter they are set to, and I might have flipped 
```

Comment:
I believe we do need to show what filter is currently set, both when they first come to the page (DatePosted DESC) and every time the sort order is changed.

Reply:
fixed